src/components/Search/style.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.article`
+  position: relative;
+  border-radius: calc(5 / 16 * 1rem);
```

Comment:
Comment here would help regarding this computation.
Also, this looks like computed at different places under this styles file. Can we create variable for the same?